chat-widget/package.json

```diff
@@ -74,9 +74,8 @@
     "webpack-cli": "^4.9.2"
   },
   "dependencies": {
-    "@fluentui/react": "^8.49.1",
     "@microsoft/omnichannel-chat-components": "1.0.1",
-    "@microsoft/omnichannel-chat-sdk": "1.3.0",
+    "@microsoft/omnichannel-chat-sdk": "^1.3.1-main.7f52e0c",
```

Comment:
Is this intentional ?

Reply:
Will change to formal version once it's released